src/liblzma/check/crc32_table.c

```diff
@@ -12,11 +12,22 @@
 
 #include "common.h"
 
+
+// FIXME: Compared to crc32_fast.c this has to check for __x86_64__ too
+// so that in 32-bit builds crc32_x86.S won't break due to a missing table.
+#if (defined(__x86_64__) && defined(__SSSE3__) \
```

Comment:
This #if needs to be:
`#if !defined(HAVE_ENCODERS) && ((defined(__x86_64__) && defined(__SSSE3__) \
			&& defined(__SSE4_1__) && defined(__PCLMUL__)) \
		|| (defined(__e2k__) && __iset__ >= 6))`

The `!defined(HAVE_ENCODERS)` part is needed because the LZ encoder needs the `lzma_crc32_table`. See lz_encoder_hash.h if you are interested.